docs/deployment-digital-ocean.md

```diff
@@ -0,0 +1,49 @@
+[deployment-digital-ocean]: /images/deployment-digital-ocean.png "Digital Ocean Dashboard "
+
+# Deploy on Digital Ocean
+
+Running Nakama in a Digital Ocean droplet is a great way to develop using Nakama without needing to install it locally.
+
+We’ll install Nakama on your Digital Ocean droplet using our standard [Docker installation guide](install-docker-quickstart.md) but first there are some things we need to configure on the droplet.
```

Comment:
The "we'll" sounds odd here for some reason to me.  Perhaps instead  "Let's get started by installing Nakama..." ?